libs/filabridge/include/private/filament/Variant.h

```diff
@@ -206,85 +212,15 @@ struct Variant {
         return Variant(key & rhs);
     }
 
+    static Variant filterVariant(
```

Comment:
I think it's confusing that we now have two `filterVariant` methods. Can you add a comment describing what each one does and/or rename one of them?

Reply:
ah yes... good point.

Reply:
done